src/api/presence.ts

```diff
@@ -100,17 +100,31 @@ export const subscribeToCheckIns: (
     queryOptions.push(fs.limit(limit));
   }
 
+  if (spaceIds.length > 10) {
+    console.error(
+      "Too many space IDs provided. Firebase limits to 10 items in an IN query. Truncating"
+    );
+    spaceIds.splice(10);
+  }
```

Comment:
[Question] I think we had an option to batch requests if the total number is larger than the max set by firebase. But I suppose we won't need it in this case?

Reply:
We could do. We'd have to loop over them and setup multiple subscriptions. Might end up getting messy/slow. Matt is happy with the limit of 10 for now :)